src/dataaccess/activity.js

```diff
@@ -0,0 +1,19 @@
+import connection from "../db/connection";
+
+const getActivities = async (req, res) => {
+  const [result] = await connection
+    .promise()
+    .query("SELECT * FROM `Actividades`");
+
+  return result;
+};
```

Comment:
Esta función no va a recibir ningún parametro, podrías dejarla así:

```suggestion
const getActivities = async () => {
  const [result] = await connection
    .promise()
    .query("SELECT * FROM `Actividades`");

  return result;
};
```